contribs/src/main/java/com/netflix/conductor/contribs/publisher/RestClient.java

```diff
@@ -0,0 +1,51 @@
+package com.netflix.conductor.contribs.publisher;
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+import org.apache.http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RestClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RestClient.class);
+    private static final String URL = "http://bullwinkle.default.svc.cluster.local:7979/v1";
```

Comment:
In the next revision, can we take this from env variable instead of hard coding here.